### DISCUSSION TITLE
请问预训练使用的中文词表和指令精调使用的指令数据集会公开吗

### DISCUSSION THREAD

**Discussion by Hugh0110** [ymcui/Chinese-LLaMA-Alpaca-2#234] [General]
没有看到您在仓库里放置数据集的链接

**Comment by jamesljl** [comment]
据说这部分是不开源的。

**Comment by ovo-Tim** [comment]
一期里似乎是有链接的，但是不是很具体。建议公开训练数据，防止增量训练时训练重复的数据。😉

**Reply by ovo-Tim** [reply]
https://huggingface.co/datasets/OpenAssistant/oasst1
https://github.com/brightmart/nlp_chinese_corpus
这是来自一期的链接
